src/index.ts

```diff
@@ -389,6 +390,12 @@ export const parse = (
         }
         reportToken(t, "unexpected token");
         return;
+      },
+      done: () => {
+        reportToken(
+          { kind: "}", position: position },
```

Comment:
Can you check how does original code catch the case of missing `}` in the middle of Json file?

Reply:
I checked the original code. The original code can't catch the case either and report no error.

Add unit test to cover the middle curly brace.

Reply:
It seems when the closing curly braces is missing within an array before closing bracket `]`. Can you please double check?

Reply:
Checked. And add unit test.